chroma_core/lib/graphql.py

```diff
@@ -0,0 +1,27 @@
+import json
+import os
+import requests
+import settings
+from django.db import connection
+
+
+def get_api_session_request():
+    cursor = connection.cursor()
+    cursor.execute("SELECT * from api_key()")
+    (API_USER, API_KEY) = cursor.fetchone()
+    cursor.close()
+    s = requests.Session()
+    s.headers.update({"AUTHORIZATION": "ApiKey {}:{}".format(API_USER, API_KEY)})
+    return s
+
+
+def graphql_query(query):
+    """
+    Issue a GraphQL query
+    """
+    s = get_api_session_request()
```

Comment:
This is going to add 1 more db connection / query per-request. This should instead by saved to a module-level variable that is first checked and if empty does this.

Reply:
okay.

Reply:
I was wondering about this as well. We also have /var/lib/chroma/iml-settings.conf that we could read from and cache. I guess if it's cached it shouldn't make much of a difference though.

Reply:
`/var/lib/chroma/iml-settings.conf` will mean introducing env vars into the python services, which shouldn't be necessary.